Mathlib/MeasureTheory/Covering/Vitali.lean

```diff
@@ -63,6 +63,10 @@ theorem exists_disjoint_subfamily_covering_enlargment (B : ι → Set α) (t : S
   the already chosen one, pick another element with large `δ`. Go on forever (transfinitely) until
   there is nothing left.
 
+@[deprecated (since := "2024-12-25")]
```

Comment:
These lines are in the middle of the proof: can you please move them to a proper place? Thanks!